sqlalchemy_jsonapi/unittests/test_serializer_get_collection.py

```diff
@@ -106,7 +106,7 @@ def test_get_collection_response_with_single_include_model(self):
                     }
                 },
                 'attributes': {
-                    'content': 'This is a comment'
+                    'content': u'This is a comment'
```

Comment:
Ok, that explanation says to me there's a bug somewhere. That shouldn't matter. I'll allow it for now, but we need to come back and work out why the lib isn't consistent in unicode/string handling. Or why our tests aren't, whatever the case may be.